lib/client.js

```diff
@@ -151,14 +151,16 @@ Client.prototype._defineService = function(service, endpoint) {
   return def;
 };
 
+var nonIdentifierChars = /[^a-z$_0-9]/i;
```

Comment:
Please, move the `var`-declaration to the top of the file, where all the other "file-wide variables" are defined.

Reply:
OK, moved it just above the constructor of `Client`.
Also fixed some JSHint warnings I introduced.